### ISSUE TITLE
[FR] Support Right To Left in all templates

### ISSUE THREAD

**Issue by khammami** [mitchelloharawild/vitae#217]
It would be awesome if you could add RTL support for all templates.

Thanks!

**Comment by mitchelloharawild** [comment]
Agreed! I'd be happy to accept pull requests for this feature. However ultimately this package aims to provide rmarkdown formats for commonly used CV templates, many of which don't support RTL. If the upstream templates support RTL, or if there is a standard/easy way to implement this then I may be able to find time for this feature.
